### PR TITLE
Add minimum required ruby version for this gem

### LINKED ISSUE #35: Specify Ruby versions supported by Skunk
Using ruby 1.8.7 😬
 
```
gem install skunk
Fetching: sexp_processor-4.14.1.gem (100%)
ERROR:  Error installing skunk:
        sexp_processor requires Ruby version ~> 2.2.
```

```
RubyGems Environment:
  - RUBYGEMS VERSION: 1.8.30
  - RUBY VERSION: 1.8.7 (2012-06-29 patchlevel 370) [x86_64-linux]
  - INSTALLATION DIRECTORY: /usr/local/lib/ruby/gems/1.8
  - RUBY EXECUTABLE: /usr/local/bin/ruby
  - EXECUTABLE DIRECTORY: /usr/local/bin
  - RUBYGEMS PLATFORMS:
    - ruby
    - x86_64-linux
  - GEM PATHS:
     - /usr/local/lib/ruby/gems/1.8
     - /root/.gem/ruby/1.8
  - GEM CONFIGURATION:
     - :update_sources => true
     - :verbose => true
     - :benchmark => false
     - :backtrace => false
     - :bulk_threshold => 1000
  - REMOTE SOURCES:
     - http://rubygems.org/
```

### PR DESCRIPTION
**Pull request by etagwerker**
Hey,

This PR fixes #35.

Please check it out.

Thanks!

### CODE REVIEW

File: skunk.gemspec [hunk 1]
```diff
@@ -14,6 +14,8 @@ Gem::Specification.new do |spec|
   spec.description   = "Knows how to calculate the StinkScore for a Ruby file"
   spec.homepage      = "https://github.com/fastruby/skunk"
 
+  spec.required_ruby_version = ">= 2.5.0"
```
Comment (bronzdoc) [inline skunk.gemspec:17]: Does the test suite breaks using `2.0.0`?

Comment (etagwerker) [reply skunk.gemspec:17]: @bronzdoc Not sure, but we're only testing with Ruby 2.5 and above. 

I'm trying to find out if it works over here: https://github.com/fastruby/skunk/pull/38/commits/5f42de978d0e7b610c6cf58df0a87bea9de7012e

Comment (etagwerker) [reply skunk.gemspec:17]: @bronzdoc Actually, trying to answer your question: https://github.com/fastruby/skunk/pull/39

Comment (bronzdoc) [reply skunk.gemspec:17]: @etagwerker I think the highest we should support is ruby >= 2.4. I updated https://github.com/fastruby/skunk/pull/39

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -1,22 +1,33 @@
...
+  Exclude:
+    - 'skunk.gemspec'
+
 # Offense count: 1
 # Cop supports --auto-correct.
 Layout/ClosingHeredocIndentation:
   Exclude:
     - 'lib/skunk/cli/commands/status_reporter.rb'
 
+Layout/IndentHeredoc:
```
Comment (bronzdoc) [inline .rubocop_todo.yml:22]: This should be deleted?

Comment (etagwerker) [reply .rubocop_todo.yml:22]: @bronzdoc If I delete it Rubocop complains a code style violation. I've followed their suggestion and renamed it as `Layout/HeredocIndentation`

**Pull request merged**